docker-compose.yml

```
@@ -0,0 +1,28 @@
+version: "3.8"
+services:
+  telegram-bot:
+    build: .
+    container_name: telegram-bot
+    ports:
+      - "8080:8080"
+    depends_on:
+      mongodb:
+        condition: service_healthy
+  mongodb:
+    image: mongo
```

```suggestion
    image: mongo : 6.0
```
Good practice to specify image's tag. Don't trust latest version